server/index.ts

```diff
@@ -123,6 +130,21 @@ const spectator = new Spectator(
 const scene = new SceneTracker();
 const boardSync = new BoardSync(updateBoard);
 
+let watchdogTimer: Watchdog;
+const watchdogFood = {
+    data: "yummy",
+    timeout: TIMEOUT,
+};
```

Comment:
is the yummy really needed here lol

Reply:
I know, I know... it causes errors without *some* data property

Reply:
the author of the library REALLY likes to simulate this to an actual dog